modules/diskq/diskq.c

```diff
@@ -178,6 +178,13 @@ _attach(LogDriverPlugin *s, LogDriver *d)
   if (self->options.prealloc < 0)
     self->options.prealloc = disk_queue_config_get_prealloc(cfg);
 
+  if (self->options.truncate_size_ratio < 1 && self->options.prealloc)
+    {
+      msg_error("Cannot disable disk-buffer truncation while preallocation is enabled. "
```

Comment:
[optional] Would it be possible to obtain which logpipe this is connected for easier diagnostics ?

Reply:
Fixed